### ISSUE TITLE
[FORMATTING] Postgres: Valid query is triggering a syntax error

### ISSUE THREAD

**Issue by acarl005** [sql-formatter-org/sql-formatter#447]
**Input data**

Which SQL and options did you provide as input?

```sql
SELECT;
```

Though counter-intuitive, this is actually a valid query in Postgres. It returns a table with 1 row and 0 columns.

**Expected Output**

```sql
SELECT;
```

**Actual Output**

```
Error: Parse error at token: ;
Unexpected DELIMITER token: {"type":"DELIMITER","raw":";","text":";","start":6,"end":7}. Instead, I was expecting to see one of the following:
# <error message truncated>
```

**Usage**

```js
format("SELECT;, {
    language: "postgresql"
})
```

**Comment by nene** [comment]
Thanks for reporting. Will be fixed in next release.

**Comment by nene** [comment]
Fixed in 10.7.2
